lowatt_grdf/api.py

```diff
@@ -214,8 +217,14 @@ def donnees_techniques(self, pce: str) -> Any:
 class StagingAPI(BaseAPI):
     scope = "/adict/bas/v4"
     api = "https://api.grdf.fr/adict/bas/v4"
+    auth_endpoint = (
+        "https://sofit-sso-oidc.grdf.fr/openam/oauth2/realms/externeGrdf/access_token"
+    )
 
 
 class API(BaseAPI):
     scope = "/adict/v2"
     api = "https://api.grdf.fr/adict/v2"
+    auth_endpoint = (
+        "https://adict-connexion.grdf.fr/oauth2/aus5y2ta2uEHjCWIR417/v1/token"
```

Comment:
This doesn't work for me, I get `{'errorCode': 'invalid_client', 'errorSummary': "Invalid value for 'client_id' parameter.", 'errorLink': 'invalid_client', 'errorId': '...', 'errorCauses': []}` but I guess this is because I need new pair of client_id/client_secret from grdf.

Does it work for you ?

Reply:
Ok I asked grdf about this, clients will be migrated by batch during several weeks. Not sure when it will ends.

So I propose to handle this by fallback to the old endpoint in case of authentication error, what do you think ?